met-api/src/met_api/resources/engagement.py

```diff
@@ -90,11 +90,6 @@ def get():
                 'created_to_date': args.get('created_to_date', None, type=str),
                 'published_from_date': args.get('published_from_date', None, type=str),
                 'published_to_date': args.get('published_to_date', None, type=str),
-                'project_type': args.get('project_type', None, type=str),
-                'project_id': args.get('project_id', None, type=str),
```

Comment:
Is it ok to remove the project_id? Is it a primary key?

Reply:
Good question! From reading the user documentation and inspecting the code, it looks like there isn't a table for projects but that the user just enters the ID manually. So no, I don't think they're a primary key.

My guess is that "projects" are domain-specific to EAO and they're managed outside of the MET app. The ticket wasn't explicit about the user of "projects" and their IDs in the Engagement version of MET so I left some instances of project_id intact until we get direction from the PO on that.